src/modules/jcms_article/jcms_article.module

```diff
@@ -32,8 +32,10 @@ function jcms_article_node_presave(EntityInterface $entity) {
     return NULL;
   }
   $node_presave = \Drupal::service('jcms_article.hooks.node_presave');
-  $node_presave->addJsonFields($entity);
-  $node_presave->setPublishedStatus($entity);
-  $node_presave->setStatusDate($entity);
-  $node_presave->setSubjectTerms($entity);
+  // TODO: retrieve with getArticleById() and pass it in
+  $article = $node_presave->getArticleById($entity->label());
+  $node_presave->addJsonFields($entity, $article);
+  $node_presave->setPublishedStatus($entity, $article);
+  $node_presave->setStatusDate($entity, $article);
+  $node_presave->setSubjectTerms($entity, $article);
```

Comment:
It seems wrong to have to keep on adding the same `$article` value. Get how we should avoid multiple requests but surely this could be handled by ensuring that we retained the results of previews requests when calling `getArticleById()`.

Perhaps we should discuss.